internal/generate/olm-catalog/package_manifest_test.go

```diff
@@ -112,63 +111,6 @@ func TestValidatePackageManifest(t *testing.T) {
 	}
 }
 
-func TestNewPackageManifest(t *testing.T) {
```

Comment:
Why remove the test? Should not it just be removed when the command be removed?

Reply:
We're not creating new package manifests anymore, so there's no point in testing their creation.